staging/src/k8s.io/kubectl/pkg/cmd/scale/scale.go

```diff
@@ -209,7 +209,7 @@ func (o *ScaleOptions) RunScale() error {
 	}
 
 	infos := []*resource.Info{}
-	err = r.Visit(func(info *resource.Info, err error) error {
+	_ = r.Visit(func(info *resource.Info, err error) error {
```

Comment:
this actually seems like a bug... I'm not sure whether we should be ignoring this error or not... can you split this into its own PR and tag @seans3 / @soltysh for a review on whether we should handle the error better or ignore it?

Reply:
Okay, will do! On side note, i think that PR will only have 1 addition then, is it fine?